Fw/Types/Assert.cpp

```diff
@@ -128,7 +128,7 @@ namespace Fw {
 
     NATIVE_INT_TYPE SwAssert(FILE_NAME_ARG file, NATIVE_UINT_TYPE lineNo) {
         if (NULL == s_assertHook) {
-            I8 assertMsg[256];
+            CHAR assertMsg[256];
```

Comment:
We should move this magic array size number to the config header.

Reply:
Different PR?

Reply:
This was fixed.